### PR TITLE
Create survival2d.is-an.app

### PR DESCRIPTION
**Pull request by tiennm99**
Add CNAME to my current personal blog

### Plan
I make it for my project in school, but current it have not done yet. So I make a CNAME point to my personal blog as a holder

### Link to Website

Link: https://miti99.eu.org

### CODE REVIEW

**Comment by tarampampam** [comment]
Wrong file name, add the `.json` at the end, please

File: survival2d.is-an.app [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My project on school",
+  "subdomain": "survival2d",
+  "owner": {
+    "repo": "https://github.com/tiennm99/free-domains",
+    "email": "minhtienit99@gmail.com"
+  },
+  "record": {
+    "CNAME": "miti99.eu.org"
```
Comment (tarampampam) [inline survival2d.is-an.app:10]:
```suggestion
    "CNAME": "miti99.eu.org."
```

**Review by tarampampam — CHANGES_REQUESTED** [review]
Change file extension

**Review by tarampampam — CHANGES_REQUESTED** [review]
The old file is not needed, right?

**Comment by tiennm99** [comment]
> The old file is not needed, right?

Yes, I don't see it in my branch so I add new file instead

**Comment by tarampampam** [comment]
Take a look into the repository root directory

**Comment by tiennm99** [comment]
> Take a look into the repository root directory

Okay, I see & delete it. Thank you

**Review by tarampampam — CHANGES_REQUESTED** [review]
The subdomain and filename did not match

**Review by tarampampam — APPROVED** [review]

**Pull request merged**